examples/sampling/graphbolt/node_classification.py

```diff
@@ -309,9 +322,9 @@ def train(args, graph, features, train_set, valid_set, num_classes, model):
 
             total_loss += loss.item()
 
-        t1 = time.time()
         # Evaluate the model.
         acc = evaluate(args, model, graph, features, valid_set, num_classes)
+        t1 = time.time()
```

Comment:
Why change this? I think we are using this log in regression test to monitor the performance.

Reply:
The time measurement did not take the validation stage into account and I thought it was misleading. When we print information about an epoch, we print the time as well and I think that it should be the whole epoch time.

Reply:
I understand that the less changes, the easier to merge so we can consider this in a separate PR.

Reply:
Took back the change.